docs/src/views/ShorthandProps.tsx

```diff
@@ -86,14 +86,12 @@ const ShorthandProps = props => (
     <blockquote>
       <strong>
         There is a very important caveat here, though: whenever React Element is directly used as a
-        shorthand value, it becomes client's responsibility to handle some aspects that Stardust was
-        originally responsible for (such as {code('styles')} or {code('accessibility')}).
+        shorthand value, all props that Stardust has created for the slot's Component will be spread
+        on the passed element. This means, you may end up with invalid props applied on HTML
```

Comment:
`This means that provided element should be able to handle Stardust props - while this requirement is satisfied for all Stardust components, you should be aware of that when either HTML or any third-party elements are provided.`

The reason I would suggest to rephrase original one is elimination of `invalid props` - in fact, there is nothing invalid in props provided, it is just about element may be not aware of necessity to handle them.